### ISSUE TITLE
Can i revoke all user's tokens by user_id?

### ISSUE THREAD

**Issue by verseal** [bshaffer/oauth2-server-php#955]
Can i revoke all user's tokens by user_id on server side?

**Comment by 0ddlyoko** [comment]
Yes, maybe revoke for specific user_id before calling `$server->verifyResourceRequest(...)`or delete token in table _oauth_authorization_codes_ or delete the user in table _oauth_clients_
